util/jsonCheck.js

```diff
@@ -1,4 +1,9 @@
 module.exports = (filename) => {
+  // Returns if filename is invalid
+  if (!filename) {
+    return;
+  }
+  // Adds a .json if the file deosnt have a .json
   if (!filename.includes('.json')) {
```

Comment:
you can remove the first if statement entirely by changing the second if statement to be:
```suggestion
  if (filename && !filename.includes('.json')) {
```